src/archiefvernietigingscomponent/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -609,15 +622,108 @@ msgstr ""
 #: archiefvernietigingscomponent/destruction/tests/test_review_create.py:170
 #: archiefvernietigingscomponent/destruction/tests/test_review_create.py:225
 #: archiefvernietigingscomponent/destruction/tests/test_review_create.py:267
-#: archiefvernietigingscomponent/destruction/views.py:414
+#: archiefvernietigingscomponent/destruction/views.py:410
 #, python-brace-format
 msgid "{author} has reviewed the destruction list."
 msgstr "{author} heeft de vernietigingslijst beoordeeld."
 
 #: archiefvernietigingscomponent/destruction/utils.py:21
+#, python-format
 msgid "%(duration)s days"
 msgstr "%(duration)s dagen geleden"
 
+#: archiefvernietigingscomponent/emails/constants.py:7
+msgid "Report available"
+msgstr ""
+
+#: archiefvernietigingscomponent/emails/constants.py:8
+msgid "Review required"
+msgstr ""
```

Comment:
```suggestion
msgstr "Review verplicht"
```

Reply:
I think that the English for this is more along the line of "Review requested", so I'm not sure about "verplicht" :thinking:

Reply:
`Review vereist`